experimental/saiportextensions.h

```diff
@@ -35,7 +35,7 @@
  */
 typedef enum _sai_port_attr_extensions_t
 {
-    SAI_PORT_ATTR_EXTENSIONS_RANGE_START = SAI_PORT_ATTR_END,
+    SAI_PORT_ATTR_EXTENSIONS_RANGE_START = 0x20000000,
```

Comment:
should we add an enum value in the saiport.h, in case this value being incorrectly used, e.g. mis-copied a 0, or value being used somewhere else for other things.

Reply:
yes

Reply:
done

Reply:
that's fast! thank you!!!